rag/advanced_rag/knowlege_compile/wiki_incremental.py

```
@@ -0,0 +1,2428 @@
+                    }
+                )
+
+        if d.get("action") == "delete":
+            entry["action"] = "delete"
+        elif entry.get("action") != "delete":
+            entry["action"] = d.get("action")
+
+    # Also enrich concept pages with entity claims from canonical_map
+    # that share source_chunk_ids with this concept's claims
+    if canonical_map:
+        for pid, entry in concept_deltas.items():
```

Ruff (0.16.0)

[warning] Loop control variable `pid` not used within loop body

(B007)